bagit.py

```diff
@@ -784,6 +784,15 @@ def _make_tagmanifest_file(alg, bag_dir):
         for digest, filename in checksums:
             tagmanifest.write('%s %s\n' % (digest, filename))
 
+def _find_tag_files(bag_dir):
```

Comment:
I think the Python community has increasingly discouraged the convention of using `_` prefixes like this, but there's precedent in the other functions in this file. I'd be inclined to remove it.